src/ncp/ncp_base.hpp

```diff
@@ -51,10 +51,10 @@
 #include "changed_props_set.hpp"
 #include "common/instance.hpp"
 #include "common/tasklet.hpp"
-#include "spinel/spinel.h"
-#include "spinel/spinel_buffer.hpp"
-#include "spinel/spinel_decoder.hpp"
-#include "spinel/spinel_encoder.hpp"
+#include "lib/spinel/spinel.h"
+#include "lib/spinel/spinel_buffer.hpp"
+#include "lib/spinel/spinel_decoder.hpp"
+#include "lib/spinel/spinel_encoder.hpp"
```

Comment:
Can we consider having `src/lib` folder in the include path? This way we won't need `lib/` when calculating the header files.

Reply:
I am trying to follow google's c++ style as a second guide if not covered by OpenThread's style guide. I prefer using a full path because it works better if there are duplicate file names inside a project.
https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes